sinks/prometheus/prometheus.go

```diff
@@ -139,3 +163,17 @@ func metricTypeEnc(metric samplers.InterMetric) string {
 	}
 	return ""
 }
+
+func MigrateConfig(conf *veneur.Config) error {
```

Comment:
nit: `MigrateConfig` always returns nil, so it should be fine to remove the `error` return type.